src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -733,6 +749,28 @@ public function collectFromCode($content, $fileName, Module $module)
                     continue;
                 }
 
+                // Allow _t(Entity.Key, 'translation', $var) and expand _t(Entity.Key, $var)
+                if ($id == T_VARIABLE && !empty($currentEntity)) {
+                    // We have a translation, eg: _t(Entity.Key, 'translation', $var)
+                    if (count($currentEntity) == 2) {
+                        continue;
+                    }
+
+                    // The variable is the second argument or present in the parameter
+                    // Try to find it _t(Entity.Key, $var)
+                    $stringValue = $stringVariables[$text] ?? null;
+
+                    // It's translated, continue
+                    if ($stringValue) {
+                        $stringValue = $this->processString($stringValue);
+                        // Deal with ''
```

Comment:
Does this mean "deal with empty strings" or "deal with single quotes"? OR something else?
And in either case, what scenario do we find those in? Is it in the first argument? The second? Somewhere else?

Reply:
it means if a string was an empty string (=> '') then there is no point in storing an empty translation